src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +28,15 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        List<String> longestWords = words.stream()
+                .filter(it -> it.length() > 12)
+                .distinct()
+                .sorted((o1, o2) -> Integer.valueOf(o2.length()).compareTo(Integer.valueOf(o1.length())))
```

Comment:
```suggestion
                .filter(word -> word.length() > 12)
                .distinct()
                .sorted((word1, word2) -> Integer.valueOf(word2.length()).compareTo(Integer.valueOf(word1.length())))
```

가독성을 위해 변수명을 타입 및 의미를 유추하기 쉬운 단어로 정해주시면 좋을 것 같습니다!

Reply:
네, 좋은 생각 입니다. 🥰